packages/funding-rate-proposer/src/proposer.js

```diff
@@ -163,8 +163,33 @@ class FundingRateProposer {
 
     // Assume pricefeed has been cached and updated prior to this function via the `update()` call.
     const priceFeed = this.priceFeedCache[fundingRateIdentifier];
-    const pricefeedPrice = await this._getPriceFeedAndPrice(fundingRateIdentifier);
-    if (!pricefeedPrice) return;
+    if (!priceFeed) {
+      this.logger.error({
+        at: "PerpetualProposer",
+        message: "Failed to create pricefeed for funding rate identifier",
+        fundingRateIdentifier
+      });
+      return null;
+    }
+    // Get hypothetical timestamp to use for a new price proposal:
+    // Unless `usePriceFeedTime=true`, use the latest block's timestamp as the request
```

Comment:
When would we want to use `usePriceFeedTime`?

Reply:
In test environments so we don't have to modify the block timing from JS. This is convenient for testing because we can manually set the pricefeed's timestamp easier than we can manually set the block time

Reply:
Ah, makes sense!